### DISCUSSION TITLE
Adding filter with a variety of filtering options

### DISCUSSION THREAD

**Discussion by dhklett** [twilio-labs/paste#3256] [Office hours] [answered]
### Background

We're adding in more filters for Ops to use when reviewing Brands and Campaigns for approval. Some of the fields will need to be open search and some have defined values to choose from.

### Feedback

I'm wondering if I'm using the side modal effectively or if there might be more useful ways I can add in a variety of new filters that users can choose to search by.

### Stage of design

In the exploration phase

### Relevant links

https://www.figma.com/file/7qKwQOUI2iJcc8B91o2WAV/Campaign%2FBrand-Details-Review-Search?type=design&node-id=0%3A1&t=WY2NIFKJ6Bf22D1o-1

### Are you looking for visual design support?

- [X] Yes

**Comment by libertycarlin** [comment, marked as answer]
Thanks for joining office hours!! Here's the recording. https://www.loom.com/share/d2bf176609ba4a899b63eb6a8c28df98
